--- pyudev-0.17/doc/conf.py~	2015-08-26 19:25:15.000000000 +0200
+++ pyudev-0.17/doc/conf.py	2015-09-15 18:16:33.788434427 +0200
@@ -103,8 +103,8 @@
 html_sidebars = {'**': ['info.html', 'localtoc.html', 'relations.html',
                         'sourcelink.html']}
 
-intersphinx_mapping = {'python': ('http://docs.python.org/', None),
-                       'pytest': ('http://pytest.org/latest', None)}
+intersphinx_mapping = {'python': ('http://docs.python.org/', 'python-objects.inv'),
+                       'pytest': ('http://pytest.org/latest', 'python-objects.inv')}
 
 
 class UDevVersion(Directive):
